src/components/lime-ai-avatar/examples/lime-ai-avatar-basic.tsx

```
@@ -0,0 +1,48 @@
+ * }
+ * ```
+ *
+ */
+@Component({
+    tag: 'limel-example-lime-ai-avatar-basic',
+    shadow: true,
+    styleUrl: 'lime-ai-avatar-basic.scss',
+})
+export class LimeAiAvatarBasicExample {
+    @State()
+    private isThinking: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)